dataduct/config/logger_config.py

```diff
@@ -22,7 +22,10 @@ def logger_configuration():
         log_directory = config.logging.get(
             'LOG_DIR', os.path.join(os.path.expanduser(CONFIG_DIR)))
         file_name = config.logging.get('LOG_FILE', LOG_FILE)
-        console_level = config.logging.get('DEBUG_LEVEL', logging.WARNING)
+        console_level = config.logging.get('CONSOLE_DEBUG_LEVEL',
```

Comment:
Should format the four statements like
```
VARIABLE_NAME = config.logging.get(
    'CONFIG_NAME', DEFAULT_VALUE)
```
so it looks consistent and more readable